x-pack/plugin/core/src/main/resources/security-index-template-7.json

```diff
@@ -213,8 +213,17 @@
               "type": "date",
               "format": "epoch_millis"
             },
-            "superseded_by": {
-              "type": "keyword"
+            "superseding_access_token": {
+              "type": "binary"
+            },
+            "superseding_refresh_token": {
+              "type": "binary"
+            },
+            "superseding_encryption_iv": {
+              "type": "binary"
+            },
+            "superseding_encryption_salt": {
+              "type": "binary"
```

Comment:
I would move `superseding_encryption_iv` and `superseding_encryption_salt` as data in the refresh token, the same way the standalone `access_tokens` were encrypted before.
The idea is to minimize the number of mapped fields, and also "hide" the implementation details.